g2p/app.py

```diff
@@ -123,30 +145,26 @@ def return_echart_data(tg: Union[CompositeTransductionGraph, TransductionGraph])
     return nodes, edges
 
 
-@APP.route("/")
-def home():
+@APP.get("/", response_class=HTMLResponse)
+def home(request: Request):
     """Return homepage of g2p studio"""
-    return render_template("index.html")
+    return TEMPLATES.TemplateResponse("index.html", {"request": request})
 
 
-@APP.route("/docs")
```

Comment:
I can never remember the whole `/api/v1/docs` path we need to get this info out of FastAPI (I usually remember the tokens, but have to try them in multiple orders before I get it right). Any chance we can `/docs` as a route that redirects to `/api/v1/docs`?

Reply:
Besides being easier to remember, it would be more backwards compat.

Reply:
Yes, this should be doable.  In fact arguably the top-level server should just use Starlette directly and not FastAPI (this would solve our fastapi-socketio issues) - then we can define whatever routes we like, including, I believe, pointing into the /docs for a mounted sub-application.  I'm curious to see if this works so I'll probably hack on it a bit this weekend.

Reply:
(to clarify: the APIs are still FastAPI - but what it gives you is just an ASGI application, which can be "mounted" anywhere you like on Starlette)

Reply:
Okay, this is done now, and also we just use Starlette directly for the top-level app since FastAPI gives us nothing there (and in fact creates problems due to fastapi-socketio).  I use redirections to point /static/swagger.json and /docs to the same things as before.